backend/app/api/ws/connection_utils.py

```
@@ -7,6 +7,50 @@
 import logging
 from typing import Any
 
+from socketio.exceptions import ConnectionRefusedError
```

Ruff (0.15.17)

[error] Import `ConnectionRefusedError` is shadowing a Python builtin

(A004)